packages/mdl-radio/index.js

```diff
@@ -0,0 +1,43 @@
+import MDLBaseComponent, {
+  MDLBaseAdapter
+} from 'mdl-base-component';
+import MDLRadioMixin, {
+  Identifier
+} from './mixin';
+import EventBus from './event-bus';
+
+const DEFAULT_EVENT_BUS = new EventBus();
+
+export default class MDLRadio extends MDLBaseComponent {
+  static attachTo(root, eventBus = DEFAULT_EVENT_BUS) {
+    const nativeInput = root.getElementsByClassName(Identifier.NATIVE_INPUT)[0];
```

Comment:
Done. We might want to remove attachTo methods in lieu of the constructor approach unless there's other potential use for them.